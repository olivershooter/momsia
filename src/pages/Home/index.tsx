import { Typography } from '@mui/material';
import React from 'react';
import FakeText from '../../components/FakeText';

const Home = () => {
  return (
    <>
      <Typography sx={{ color: 'black' }}>home</Typography>
      <FakeText />
    </>
  );
};

export default Home;
