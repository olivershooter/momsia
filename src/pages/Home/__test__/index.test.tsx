import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import { describe, expect, it } from 'vitest';
import '@testing-library/jest-dom';
import Home from '..';

import { render, screen } from '@testing-library/react';

describe('Home page', () => {
  it('renders and matches the snapshot', () => {
    const { asFragment } = render(
      <BrowserRouter>
        <Home />
      </BrowserRouter>,
    );

    expect(asFragment()).toMatchSnapshot();
  });

  it('finds the text home', () => {
    render(
      <BrowserRouter>
        <Home />
      </BrowserRouter>,
    );

    expect(screen.getByText('lol')).toBeInTheDocument();
  });
});
