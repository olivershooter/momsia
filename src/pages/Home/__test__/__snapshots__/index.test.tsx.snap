// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Home page > renders and matches the snapshot 1`] = `
<DocumentFragment>
  <p
    class="MuiTypography-root MuiTypography-body1 css-nnsfc0-MuiTypography-root"
  >
    home
  </p>
</DocumentFragment>
`;
